cfgov/cfgov/settings/base.py

```diff
@@ -152,6 +152,11 @@
 
 ALLOWED_HOSTS = ['*']
 
+#Enabling compression for use in base.html
+COMPRESS_ENABLED = True
```

Comment:
I like it as a file because I can then use https://chrome.google.com/webstore/detail/pretty-beautiful-javascri/piekbefgpgdecckjcpffhnacjflfoddg.

@anselmbradford  what do you think?